content/cdi-iii/0010-serie-fourier.md

```diff
@@ -49,29 +50,31 @@ $$
 b_n = \frac{1}{L} \int_{-L}^{L} f(x) \sin\left(\frac{n\pi x}{L}\right) \d x
 $$
 
+Note-se que na expressão acima passámos o termo $n=0$ para fora da série e trocamos $a_0$ por $\frac{a_0}{2}$. Isto é feito para que a fórmula do $a_n$ se aplique também ao $a_0$.
+
 :::
 
-A expressão resultante, $SF_f(x)$ vai corresponder exatamente a $f$ no intervalo $]-L, L[$, exceto onde a função tem pontos de descontinuidade.  
-Aliás, uma das grandes vantagens da Série de Fourier é que a função não precisa de ser contínua, apenas seccionalmente contínua (e ter derivada seccionalmente contínua).
+A expressão resultante, $SF_f(x)$ vai corresponder exatamente a $f$ nos pontos em $]-L, L[$ em que $f$ é contínua.
+Aliás, uma das grandes vantagens da Série de Fourier é que a função não precisar de ser contínua, apenas seccionalmente contínua (e ter derivada seccionalmente contínua).
 
-Nos pontos de descontinuidade e nas "pontas" do intervalo ($-L$ e $L$), a expressão da Série de Fourier equivale ao ponto médio entre os dois pontos adjacentes. Traduzindo isto para um sistema, obtemos o [**Teorema da Convergência Pontual da Série de Fourier**](color:green):
+Nos pontos de descontinuidade, a expressão da Série de Fourier equivale ao ponto médio entre os valores da função a vir de ambas as direções. Traduzindo isto para um sistema, obtemos o [**Teorema da Convergência Pontual da Série de Fourier**](color:green):
```

Comment:
@Calhau18 why did you remove `e nas "pontas" do intervalo ($-L$ e  $L$)`? Seems like you've removed key information

Reply:
If I recall correctly, it's because the edges (L and -L) belong to the function domain and therefore are either continuity points or discontinuity points. 
Remember that Fourier Series are (because they are periodic) always defined in the whole real domain.
In either case, the theorem already specifies the series' value in such points, making this redundant information. 
If you consider however that it is worth keeping, even if as a parenthesis, please do. I think that probably is the right call.